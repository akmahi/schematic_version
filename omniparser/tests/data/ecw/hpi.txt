apuid|RecordID|PatientId|PatientAccountNo|encounterId|EncounterDate|ProviderNPI|Ecwstructid|Question|QuestionID|Answers|AnswerID|ItemName|ItemID|CategoryName|CategoryID|MainCategoryName|MainCategoryID
305922|305922.532994.2473498.47|532994|532994|2473498|2022-10-17|1427131036|44|Feeling down, depressed, or hopeless|47|Yes||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.532989.2473511.47|532989|532989|2473511|2022-10-17|1427131036|44|Feeling down, depressed, or hopeless|47|Yes||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.532996.2514390.47|532996|532996|2514390|2022-10-17|1427131036|44|Feeling down, depressed, or hopeless|47|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.533068.2457682.47|533068|533068|2457682|2022-10-17|1427131036|44|Feeling down, depressed, or hopeless|47|Yes||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.529934.2473575.47|529934|529934|2473575|2022-10-17|1427131036|44|Feeling down, depressed, or hopeless|47|Yes||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.9321.2517560.47|9321|100047|2517560|2022-10-17|1679623896|44|Feeling down, depressed, or hopeless|47|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.535986.2550354.47|535986|535986|2550354|2022-10-17|1235225871|44|Feeling down, depressed, or hopeless|47|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.12749.2471657.47|12749|516477|2471657|2022-10-18|1205940046|44|Feeling down, depressed, or hopeless|47|Yes||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.532994.2473498.46|532994|532994|2473498|2022-10-17|1427131036|43|Little interest or pleasure in doing things|46|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.532989.2473511.46|532989|532989|2473511|2022-10-17|1427131036|43|Little interest or pleasure in doing things|46|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.532996.2514390.46|532996|532996|2514390|2022-10-17|1427131036|43|Little interest or pleasure in doing things|46|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.529934.2473575.46|529934|529934|2473575|2022-10-17|1427131036|43|Little interest or pleasure in doing things|46|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.9321.2517560.46|9321|100047|2517560|2022-10-17|1679623896|43|Little interest or pleasure in doing things|46|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.535986.2550354.46|535986|535986|2550354|2022-10-17|1235225871|43|Little interest or pleasure in doing things|46|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.12749.2471657.46|12749|516477|2471657|2022-10-18|1205940046|43|Little interest or pleasure in doing things|46|No||PHQ-2 In last two weeks have you been bothered by|20534|Depression Screening|234469|HPI|93
305922|305922.546423.2509021.44|546423|546423|2509021|2022-10-17|1871851568|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.15236.2542901.44|15236|524609|2542901|2022-10-17|1871851568|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.15808.2489735.44|15808|514345|2489735|2022-10-17|1205940046|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.535090.2545195.44|535090|535090|2545195|2022-10-17|1871851568|45|Little interest or pleasure in doing things|44|Several days|166|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.532066.2496409.44|532066|532066|2496409|2022-10-17|1871851568|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.526416.2521935.44|526416|526416|2521935|2022-10-17|1871851568|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.534136.2526658.44|534136|534136|2526658|2022-10-17|1518170364|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.532994.2473498.44|532994|532994|2473498|2022-10-17|1427131036|45|Little interest or pleasure in doing things|44|Several days|166|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.532989.2473511.44|532989|532989|2473511|2022-10-17|1427131036|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.533068.2457682.44|533068|533068|2457682|2022-10-17|1427131036|45|Little interest or pleasure in doing things|44|More than half the days|167|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.529934.2473575.44|529934|529934|2473575|2022-10-17|1427131036|45|Little interest or pleasure in doing things|44|Several days|166|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.9977.2517824.44|9977|400317|2517824|2022-10-17|1679623896|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.9321.2517560.44|9321|100047|2517560|2022-10-17|1679623896|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
305922|305922.552089.2533209.44|552089|552089|2533209|2022-10-17|1396385019|45|Little interest or pleasure in doing things|44|Not at all|165|PHQ-9|20535|Depression Screening|234469|HPI|93
