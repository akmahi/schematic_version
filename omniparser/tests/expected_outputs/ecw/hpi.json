[
	{
		"hpi": {
			"answers": "Yes",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473498"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532994",
				"id": "532994"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.532994.2473498.47"
		}
	},
	{
		"hpi": {
			"answers": "Yes",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473511"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532989",
				"id": "532989"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.532989.2473511.47"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-17",
				"id": "2514390"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532996",
				"id": "532996"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.532996.2514390.47"
		}
	},
	{
		"hpi": {
			"answers": "Yes",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-17",
				"id": "2457682"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "533068",
				"id": "533068"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.533068.2457682.47"
		}
	},
	{
		"hpi": {
			"answers": "Yes",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473575"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "529934",
				"id": "529934"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.529934.2473575.47"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-17",
				"id": "2517560"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "100047",
				"id": "9321"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.9321.2517560.47"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-17",
				"id": "2550354"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "535986",
				"id": "535986"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1235225871"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.535986.2550354.47"
		}
	},
	{
		"hpi": {
			"answers": "Yes",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "44",
			"encounter": {
				"date": "2022-10-18",
				"id": "2471657"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "516477",
				"id": "12749"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1205940046"
			},
			"question": "Feeling down, depressed, or hopeless",
			"question_id": "47",
			"record_id": "305922.12749.2471657.47"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "43",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473498"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532994",
				"id": "532994"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "46",
			"record_id": "305922.532994.2473498.46"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "43",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473511"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532989",
				"id": "532989"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "46",
			"record_id": "305922.532989.2473511.46"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "43",
			"encounter": {
				"date": "2022-10-17",
				"id": "2514390"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532996",
				"id": "532996"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "46",
			"record_id": "305922.532996.2514390.46"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "43",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473575"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "529934",
				"id": "529934"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "46",
			"record_id": "305922.529934.2473575.46"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "43",
			"encounter": {
				"date": "2022-10-17",
				"id": "2517560"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "100047",
				"id": "9321"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "46",
			"record_id": "305922.9321.2517560.46"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "43",
			"encounter": {
				"date": "2022-10-17",
				"id": "2550354"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "535986",
				"id": "535986"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1235225871"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "46",
			"record_id": "305922.535986.2550354.46"
		}
	},
	{
		"hpi": {
			"answers": "No",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "43",
			"encounter": {
				"date": "2022-10-18",
				"id": "2471657"
			},
			"item_id": "20534",
			"item_name": "PHQ-2 In last two weeks have you been bothered by",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "516477",
				"id": "12749"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1205940046"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "46",
			"record_id": "305922.12749.2471657.46"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2509021"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "546423",
				"id": "546423"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1871851568"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.546423.2509021.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2542901"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "524609",
				"id": "15236"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1871851568"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.15236.2542901.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2489735"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "514345",
				"id": "15808"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1205940046"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.15808.2489735.44"
		}
	},
	{
		"hpi": {
			"answer_id": "166",
			"answers": "Several days",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2545195"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "535090",
				"id": "535090"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1871851568"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.535090.2545195.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2496409"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532066",
				"id": "532066"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1871851568"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.532066.2496409.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2521935"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "526416",
				"id": "526416"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1871851568"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.526416.2521935.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2526658"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "534136",
				"id": "534136"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1518170364"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.534136.2526658.44"
		}
	},
	{
		"hpi": {
			"answer_id": "166",
			"answers": "Several days",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473498"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532994",
				"id": "532994"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.532994.2473498.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473511"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "532989",
				"id": "532989"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.532989.2473511.44"
		}
	},
	{
		"hpi": {
			"answer_id": "167",
			"answers": "More than half the days",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2457682"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "533068",
				"id": "533068"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.533068.2457682.44"
		}
	},
	{
		"hpi": {
			"answer_id": "166",
			"answers": "Several days",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2473575"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "529934",
				"id": "529934"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1427131036"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.529934.2473575.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2517824"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "400317",
				"id": "9977"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.9977.2517824.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2517560"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "100047",
				"id": "9321"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.9321.2517560.44"
		}
	},
	{
		"hpi": {
			"answer_id": "165",
			"answers": "Not at all",
			"category_id": "234469",
			"category_name": "Depression Screening",
			"ecw_struct_id": "45",
			"encounter": {
				"date": "2022-10-17",
				"id": "2533209"
			},
			"item_id": "20535",
			"item_name": "PHQ-9",
			"main_category_id": "93",
			"main_category_name": "HPI",
			"patient": {
				"account_no": "552089",
				"id": "552089"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1396385019"
			},
			"question": "Little interest or pleasure in doing things",
			"question_id": "44",
			"record_id": "305922.552089.2533209.44"
		}
	}
]
